### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by vedant-z**

### CODE REVIEW

**Comment by vedant-z** [comment]
@kishanrajput23 If you can please merge this PR on or after 1st October 😊

**Review by kishanrajput23 — CHANGES_REQUESTED** [review]
Please add source code also.

**Comment by vedant-z** [comment]
Yes I am gonna add the source code after you merge the PR so that Tic Tac Toe folder is created and then I will add the source code in that

**Pull request closed without merging**
